docs/install.md

```diff
@@ -122,34 +109,13 @@ Launching from Seqera Platform provides you with:
 Seqera Cloud Basic is free for small teams. Researchers at qualifying academic institutions can apply for free access to Seqera Cloud Pro.
 See the [Seqera Platform documentation](https://docs.seqera.io/platform) for set-up information and tutorials to get started.
 
-## Stable and edge releases
-
-A *stable* version of Nextflow is released every six months, in the 4th and 10th month of each year.
-
-Additionally, an *edge* version is released on a monthly basis. The edge releases can be used to access the latest updates and experimental features.
-
-To use the latest edge release, set `NXF_EDGE=1` when updating:
-
-```{code-block} bash
-:class: copyable
-NXF_EDGE=1 nextflow self-update
-```
-
-You can also use `NXF_VER` to temporarily switch to any edge release. For example:
-
-```{code-block} bash
-:class: copyable
-NXF_VER=24.06.0-edge nextflow info
-```
-
 ## Standalone distribution
 
 The Nextflow standalone distribution (i.e. the `dist` distribution) consists of self-contained `nextflow` executable file
-that includes all the application dependencies for core functionalities, and it can run without downloading third parties
+that includes all the application dependencies for core functionalities and can run without downloading third parties
 libraries. This distribution is mainly useful for offline environments.
 
-Note however the support for cloud services e.g. AWS, Seqera Platform, Wave, etc. still require the download
-of the corresponding Nextflow plugins.
+Support for cloud services, such as AWS, Seqera Platform, and Wave, still require downloading the corresponding Nextflow plugins.
```

Comment:
```suggestion
The Nextflow standalone distribution (i.e. the `dist` release) is a self-contained `nextflow` executable that can run without needing to download core dependencies at runtime. This distribution is useful for offline environments, as well as building and testing Nextflow locally.

The standalone distribution will still download core and third-party plugins as needed at runtime.
```

Feel free to re-phrase, but just wanted to clarify the behavior of the `dist` release.